packages/front-end/components/Experiment/Results.tsx

```diff
@@ -176,6 +180,11 @@ const Results: FC<{
     );
   }
 
+  // cannot re-aggregate quantile metrics across pre-computed dimensions
+  const showQuantileReaggregationError = analysis?.settings?.dimensions.some(
```

Comment:
Not sure what's a better name but this confused me initially because it felt we would show the error by default if any dimension is precomputed for the snapshot, which is partially true because this will only affect for quantile metrics.